docs/en/enterprise-edition/content-collections/governance/custom-build-policies/custom-build-policy-examples.adoc

```diff
@@ -12,7 +12,7 @@ To create code-based policies for your infrastructure, use these examples as gui
 [#bicep-example]
 === Bicep Example
 
-The following example demonstrates how to create a Bicep policy.
+The following example demonstrates how to create a Bicep policy. This policy ensures that auditing is enabled for Microsoft SQL servers and their databases. It filters resources to include only Microsoft.Sql/servers and Microsoft.Sql/servers/databases, then checks for a connection with auditing settings for both resource types and that the settings are enabled.
```

Comment:
The following example shows how to create a Bicep policy that ensures auditing is enabled for Microsoft SQL servers and their databases. It filters resources to include only Microsoft.Sql/servers and Microsoft.Sql/servers/databases, then verifies that auditing settings are configured and enabled for both resource types.